### PR TITLE
first stab at "what is ultimate page"

### PR DESCRIPTION
**Pull request by rnegash**

### CODE REVIEW

File: src/components/FooterSection.astro [hunk 1]
```diff
@@ -8,7 +8,7 @@
           class="footer-link"
           target="_blank"
           rel="noopener noreferrer"
-          href="https://svbvb49.de/frisbee">SV BVB 49 e.V.</a
+          href="https://www.svbvb49.de/abteilungen/frisbee">SV BVB 49 e.V.</a
```
Comment (rnegash) [inline src/components/FooterSection.astro:11]: link was broken

File: src/components/HeaderSection.astro [hunk 2]
```diff
@@ -4,23 +4,40 @@ import { Image } from "astro:assets";
...
-  trailingLinks: {
-    href: string;
-    label: string;
-  }[];
 }
 
-const { id, logoHref, trailingLinks } = Astro.props;
+const { id } = Astro.props;
+
+const trailingLinks = [
```
Comment (rnegash) [inline src/components/HeaderSection.astro:11]: Thought it made sense to just keep these here 4 consistency. yes, no?

File: src/pages/ultimate.astro [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+      <p>
+        <strong>Spirit of the Game-</strong> Ultimate relies upon a Spirit of the
+        Game that places the responsibility for fair play on every player. There
+        are no referees; the players are solely responsible for following and enforcing
+        the rules, even at World Championship. Competitive play is encouraged, but
+        never at the expense of respect between players, adherence to the rules,
+        and the basic joy of play.
+      </p>
+
+      <p>Source: <a href="https://wfdf.sport/disciplines/ultimate/">WFDF</a></p>
```
Comment (rnegash) [inline src/pages/ultimate.astro:55]: thanks wfdf

File: src/pages/ultimate.astro [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+        height="315"
+        src="https://www.youtube.com/embed/hXArTyZcjPA?si=07KMHaIEVYeDnNCD"
+        title="YouTube video player"
+        frameborder="0"
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+        referrerpolicy="strict-origin-when-cross-origin"
+        allowfullscreen></iframe>
+      <h2>
+        Ultimate is a non-contact, self-refereed team sport played with a flying
+        disc frisbee.
```
Comment (rnegash) [inline src/pages/ultimate.astro:31]:
```suggestion
https://github.com/user-attachments/assets/015cfc9c-11cb-421f-ba34-7c2367234a01

```

File: src/pages/ultimate.astro [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+<Layout title="Lobstars Ultimate Frisbee - Berlin">
+  <main class="h-full w-full">
+    <HeaderSection id="header" />
+
+    <section class="section-container grid gap-4 pt-32">
+      <h1>What is Ultimate Frisbee?</h1>
+
+      <iframe
+        width="560"
+        height="315"
```
[the remark this answers is not in the document]
Comment (rnegash) [reply src/pages/ultimate.astro:22]: Good point. Trying this solution: https://nikitahl.com/responsive-youtube-embed

File: src/env.d.ts [hunk 6]
```diff
@@ -1 +1,2 @@
+/// <reference path="../.astro/types.d.ts" />
```
Comment (rnegash) [inline src/env.d.ts:1]: Committed this by mistake, got autogenerated on my side for some reason. Not sure if its a big deal to keep it out?

**Review by karlkristopher — APPROVED** [review]

**Pull request merged**